### DISCUSSION TITLE
Error in department stats in dashboard

### DISCUSSION THREAD

**Discussion by vinruo** [LiveHelperChat/livehelperchat#2077] [Q&A]
In my dashboard, the department stats widget does not display the departments
![dep](https://github.com/LiveHelperChat/livehelperchat/assets/166167832/ae066153-90c8-49cd-851a-918b2502108c)

**Comment by remdex** [comment]
Either you have not assigned departments to yourself or do not have proper permissions.
https://doc.livehelperchat.com/docs/dashboard#departments-stats

**Comment by vinruo** [comment]
I did a new installation but the department still doesn't appear even though the user has the permissions and is assigned to all departments. However, I noticed that if I add a department group, the display is correct (both the group and the department appear), if I delete the department group, nothing appears anymore.
![withwithout](https://github.com/LiveHelperChat/livehelperchat/assets/166167832/2aa0d82e-b5ce-47e2-8cce-44ad78050aa0)

**Comment by vinruo** [comment]
The same issue is also present on the demo. https://demo.livehelperchat.com/site_admin/

**Comment by remdex** [comment]
I'll test tomorrow

**Comment by remdex** [comment]
Fix was deployed - https://github.com/LiveHelperChat/livehelperchat/commit/ca1856aa765cf58e9745dc1a04c0444f6ebf06e2

You can update those three files and click clear cache or you can wait for next official release :)
